Gateway/Http/Client/TransactionRefund.php

```diff
@@ -77,15 +76,15 @@ public function placeRequest(TransferInterface $transferObject): array
             $requestOptions['idempotencyKey'] = $idempotencyKey;
 
             $this->adyenHelper
-                ->logRequest($request, Client::API_PAYMENT_VERSION, '/pal/servlet/Payment/{version}/refund');
+                ->logRequest($request, Client::API_CHECKOUT_VERSION, '/refunds');
             try {
-                $response = $service->refund($request, $requestOptions);
+                $response = $service->refunds($request, $requestOptions);
 
                 // Add amount original reference and amount information to response
-                $response[self::REFUND_AMOUNT] = $request['modificationAmount']['value'];
-                $response[self::REFUND_CURRENCY] = $request['modificationAmount']['currency'];
+                $response[self::REFUND_AMOUNT] = $request['amount']['value'];
+                $response[self::REFUND_CURRENCY] = $request['amount']['currency'];
 
-                $response[self::ORIGINAL_REFERENCE] = $request['originalReference'];
+                $response[self::ORIGINAL_REFERENCE] = $request['paymentPspReference'];
```

Comment:
```suggestion
                $response[self::ORIGINAL_REFERENCE] = $request[TransactionCapture::ORIGINAL_REFERENCE];
```

Reply:
I would personally rather keep the paymentPspReference in the cases of multiple refunds. But lets discuss.